x/evm/handler.go

```diff
@@ -55,12 +56,13 @@ func HandleMsgEthereumTx(ctx sdk.Context, k Keeper, msg types.MsgEthereumTx) sdk
 		Payload:      msg.Data.Payload,
 		Csdb:         k.CommitStateDB.WithContext(ctx),
 		ChainID:      intChainID,
-		THash:        &txHash,
+		THash:        &ethHash,
 		Simulate:     ctx.IsCheckTx(),
 	}
+
 	// Prepare db for logs
 	// TODO: block hash
-	k.CommitStateDB.Prepare(txHash, txHash, k.TxCount)
+	k.CommitStateDB.Prepare(ethHash, common.Hash{}, k.TxCount)
```

Comment:
can we remove the second field from the `Prepare` function signature if it's not going to be used?

Reply:
it's matching the Geth interface, and will be used in the future (for adding block hash for the transactions logs)